From e3c8ec6c36c37edc10ed6077ac24c71c6734b536 Mon Sep 17 00:00:00 2001
From: Charles Stevens <cstevens@beaconembedded.com>
Date: Tue, 18 Jan 2022 12:46:35 -0600
Subject: [PATCH 61/63] - Add SPI chipselect

Add second SPI chip select to the spidev1 in the device tree.

Signed-off-by: Charles Stevens <cstevens@beaconembedded.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
index 631fa32b2e6e..d8300ecf7ccc 100755
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -752,13 +752,19 @@
 	pinctrl-0 = <&msiof1_pins>;
 	pinctrl-names = "default";
 	status = "okay";
-	cs-gpios = <&gpio3 10 GPIO_ACTIVE_LOW>;
+	cs-gpios = <&gpio3 10 GPIO_ACTIVE_LOW>, <&gpio3 11 GPIO_ACTIVE_LOW>;
 
 	spidev1@0 {
 	       compatible = "spidev";
 	       spi-max-frequency = <8000000>;
 	       reg = <0>;
 	};
+
+	spidev1@1 {
+	       compatible = "spidev";
+	       spi-max-frequency = <8000000>;
+	       reg = <1>;
+	};
 };
 
 &msiof2 {
-- 
2.17.1

